config/baseband_archiver.yaml

```diff
@@ -0,0 +1,2063 @@
+##########################################
+#
+# baseband_archiver.yaml
+#
+##########################################
+num_elements: 2048
+sampling_rate: 2
+fft_length: 2048
+nyquist_zone: 2
+samples_per_data_set: 49152
```

Comment:
Do we actually need this value?  Or is it incorrect?  The frames coming from the x-engine are 512 samples long because that's what the transfer buffer uses.  (The primary buffer is 49152, but that doesn't have any impact on the baseband system down stream).

Reply:
Actually, we don't need this file at all. I had added it because we had imagined not using the realtime Kotekan config for this system at all. Since we are currently still using the realtime config, I will remove this and we can revisit this later.

Reply:
Ah perfect, yes let's just remove this file then.